lib/jira/util/index.js

```diff
@@ -1,6 +1,19 @@
-const { markdown } = require('markdown');
-
 module.exports = function (jiraClient) {
+  const checkForReferenceText = (text) => {
```

Comment:
Can you add a comment to this function, explaining what it does and what it returns? Ideally with an example.